home-manager/flake.nix

```diff
@@ -16,7 +16,7 @@
   };
 
   outputs =
-    { nixpkgs, home-manager, ... }:
+    { nixpkgs, home-manager, ... }@inputs:
```

Comment:
I believe we will want to prefer the prefix version of this `inputs @ {` as opposed to suffix.